sofagym/rpc_server.py

```diff
@@ -525,7 +525,7 @@ def start_scene(config, nb_actions):
     # Run the first client
     def deferredStart():
         sdict = str(config)
-        subprocess.run([config['python_version'], path+"common/rpc_client.py", sdict, str(nb_actions), str(port_rpc)],
+        subprocess.run([config['python_version'], path+"sofagym/rpc_client.py", sdict, str(nb_actions), str(port_rpc)],
```

Comment:
suggestion for another PR: modifier config["python_version"] par sys.executable pour sortir la version de python utilisé de la config.